tests/subsys/pm/power_states/pytest/test_power_states.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2024 Intel Corporation.
+# SPDX-License-Identifier: Apache-2.0
+from current_analyzer import current_RMS
+
+def test_power_states(current_measurement_output):
+    data = current_measurement_output
+    # Empirical RMS values in mA with descriptive keys
+    rms_expected = {
+        "k_cpu_idle":57.0,  # K_cpu_idle
```

Comment:
ca we move those expected data to a yaml file for different platform?

Reply:
Of course we can, the good hint. Thank you.

Reply:
PS: Btw, Please take a look. https://github.com/zephyrproject-rtos/zephyr/issues/80989. Feel free to share ideas or warries. :)